cmake_configure.cmake

```diff
@@ -21,6 +22,12 @@ elseif(CMAKE_CXX_COMPILER_ID STREQUAL "MSVC")
             "_SCL_SECURE_NO_WARNINGS"
             "_CRT_SECURE_NO_WARNINGS"
         )
+    if(MSVC_PARALLEL_BUILD)
+        target_compile_options(rw_interface
+            INTERFACE
+                "/MP"
```

Comment:
Does this need to be configurable? Can we put it in the default flags above.

Reply:
It was a convenience for me to see what source emitted the bullet warnings. These warnings messages did not contain any reference to what source included them. But I guess its use is gone.